docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -6,7 +6,27 @@ description: Describes common issues and solutions for access to servers.
 This section describes common issues that you might encounter in managing access to servers
 with Teleport and how to work around or resolve them.
 
-## Starting SSH sessions fails
+## Starting SSH sessions fails with expired certificates
+
+A ssh session fails to start on certain SSH servers with the error message
```

Comment:
- SSH, not ssh
- _An SSH_, not "A SSH"
- I don't think this is a complete sentence